src/screen.js

```diff
@@ -0,0 +1,65 @@
+
+import config from './config'
+
+// Collection of components that contains `$screen` helper usecase
+const screenComponents = []
+
+// Debounce `updateElements` method to prevent memory leak
+const debouncedScreensUpdate = () => {
+    let update
+
+    // Update component if $el is in `screenComponents`
+    const updateScreens = () => {
+        clearTimeout(update)
+        update = setTimeout(() => {
+            screenComponents.forEach(($el) =>
+                $el && $el.__x && $el.__x.updateElements($el),
```

Comment:
Did we want to remove falsy `$el` before doing the forEach @SimoTod ?

I think `$el`'s are the root elements of Alpine components so the only situation I can think where we might lose some is if using nested Alpine components inside of an `x-if` or something

Reply:
Are they going to be falsy or are they just going to be detached from the dom?
Other libraries (i.e. Turbolinks) may remove those elements from the Dom too. I think we should remove it from the list in that case.